docs/Coding-Guidelines/VisualStudio2022.vsconfig

```diff
@@ -37,20 +35,16 @@
     "Component.Microsoft.VisualStudio.LiveShare.2022",
     "Microsoft.VisualStudio.Component.IntelliCode",
     "Microsoft.VisualStudio.Component.ClassDesigner",
-    "Microsoft.VisualStudio.Component.CodeClone",
     "Microsoft.VisualStudio.Component.GraphDocument",
     "Microsoft.VisualStudio.Component.CodeMap",
-    "Microsoft.VisualStudio.Component.DependencyValidation.Enterprise",
     "Microsoft.VisualStudio.Component.VC.CoreIde",
-    "Microsoft.VisualStudio.Component.Windows10SDK",
```

Comment:
I see Windows10SDK.19041 below. What's this, vLatest?

Reply:
That's my guess too, 19041 exists in both the original and new version, its seems this version is specifically called for.